morph_tool/utils.py

```diff
@@ -2,6 +2,8 @@
 from functools import partial
 from pathlib import Path
 
+import pandas as pd
```

Comment:
what is satisfying the pandas dependency?

Reply:
It was brought on by BluePyOpt. I have explicitly added the Pandas dependency in https://github.com/BlueBrain/morph-tool/pull/13